doc/template/simple_form.md

```diff
@@ -46,6 +47,29 @@ Using the [Neat Grid](http://neat.bourbon.io/) to align label and corresponding
     = f.button :submit
 ```
 
+### Field Sizing
+
+As some fields do not require an entire line to themselves (e.g. Postcode), there are 3 different row styles to choose from:
+
+* 2-1
+* 1-2
+* 2-2
+
+Each number refers to the column size input fields can assume in a grid of 7 available columns. The remainder is used for labels. When choosing the sizing, label & input value length should be considered.
+
+**Usage:**
+
+* Extra `.row` and `.sized-[x]-[y]` class for the wrapper of the 2 fields
+
+```haml
+.row.sized-2-1
+  = form.input :suburb
+  = form.input :postcode
+.row.sized-2-2
```

Comment:
@pzi explained the reasons in person.

The usage of:

```
.my-semantic-class
  @extend .row
  @extend .sized-2-2
```

Causes the huge selector chains he mentioned above.

He also mentioned this is not an issue for web awards either.

So, let's go for it.